spec/spec.md

```diff
@@ -1477,6 +1527,35 @@ The following JSON Schema Draft 7 definition summarizes the rules above:
 
 </tab-panels>
 
+## Credential Format Designations
+
+Within the _Presentation Exchange_ specification, there are numerous sections 
+where Verifiers and Holders convey what credential variants they support and 
+are submitting. The following are the normalized references used within the 
+specification:
+
+- `jwt` - the format is a [JSON Web Token](https://tools.ietf.org/html/rfc7797) 
+  that will be submitted in the form of a JWT encoded string. Expression of 
+  supported algorithms in relation to this format ****MUST**** be conveyed using an `alg` property 
+  paired with values that are identifiers from the 
+  [RFC 7518 JSON Web Algorithms](https://tools.ietf.org/html/rfc7518) registry.
+- `jwt-vc`, `jwt-vp` - these formats are [JSON Web Tokens](https://tools.ietf.org/html/rfc7797) 
+  that will be submitted in the form of a JWT encoded string, and the body of the decoded 
+  JWT string is defined in the [JSON Web Token section](https://www.w3.org/TR/vc-data-model/#json-web-token) 
+  of the W3C Verifiable Credentials specification. Expression of supported algorithms in 
+  relation to these formats ****MUST**** be conveyed using an `alg` property paired with values that 
+  are identifiers from the [RFC 7518 JSON Web Algorithms](https://tools.ietf.org/html/rfc7518) registry. 
+- `vc`, `vp` - these formats are [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) 
```

Comment:
why are we conflating the JSONLD format of VCs and VPs with the proof algorithms and associated data structures?

Reply:
I believe this comment is resolved assuming we use `proof_type`.